customer-service/src/main/kotlin/cz/kss/proj/clientservice/config/route/CustomerRouter.kt

```diff
@@ -0,0 +1,25 @@
+package cz.kss.proj.clientservice.config.route
+
+import cz.kss.proj.clientservice.handler.CustomerHandler
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.MediaType
+import org.springframework.http.MediaType.APPLICATION_JSON
+import org.springframework.web.reactive.function.server.coRouter
+
+@Configuration
+class CustomerRouter(
+    private val customerHandler: CustomerHandler,
+) {
+    @Bean
+    fun apiRouter() = coRouter {
+        "/api/customer".nest {
+            accept(APPLICATION_JSON).nest{
+                GET("",)
```

Comment:
I wonder if the nesting is right, shouldn't the `POST` be a level up?

Reply:
I fix it